places/models.py

```diff
@@ -1,3 +1,6 @@
 from django.db import models
 
 # Create your models here.
+class Artist(models.Model):
+    name = models.CharField(max_length=200)
+    website = models.URLField(blank=True)
```

Comment:
Please add a `__str__` method for rendering the artist object as a string

Reply:
@rlconley have updated this code